endcoronavirus/endcoronavirus.org
# Add /updates and /guidelines pages

This adds the `/updates` index page, and all the existing updates posts themselves (although there'll of course be another one to copy over tomorrow).

I removed the "_posts/blog" and "_posts/updates" folders, because endcoronavirus.org doesn't seem to have a blog, so I think we should just keep it simple and make all posts updates.

I created a new layout, `update`, and deleted the `blog` layout. In this layout I've added the style `p {max-width: auto}`. This is because all the updates consist of entirely images, and I don't think these images should be squeezed to `40em`, as they need to be readable.

I've removed the introductory comments from `_config.yaml`, fixed the title, improved the way titles display in `_includes/head.html`.

I added `PORT=8076`, 'cos I was getting clashes on port `8080`.

fixes #6
fixes #7

QA
--

`./run`

Go to http://localhost:8076/updates, check all the updates pages exist, work, are the same as on https://www.endcoronavirus.org/page/updates.

Go to http://localhost:8076/guidelines, check it looks ok, same content as https://www.endcoronavirus.org/page/guidelines.
